src/usr/local/www/firewall_nat_out.php

```diff
@@ -244,7 +244,8 @@
 }
 
 if (is_subsystem_dirty('natconf')) {
-	print_info_box_np(gettext("The NAT configuration has been changed.")."<br />".gettext("You must apply the changes in order for them to take effect."));
+	print_info_box_np(gettext('The NAT configuration has been changed.') . '<br />' .
+					  gettext('You must apply the changes in order for them to take effect.'));
```

Comment:
Unfortunately part of style was lost during the big bootstrap merge. the style guide is still the same, TAB with size of 8 and 4 spaces for multiple line statements